### DISCUSSION TITLE
关于管理安全问题

### DISCUSSION THREAD

**Discussion by 9sr** [AlistGo/alist#765] [Ideas]
能不能增加在后台手动设置管理入口功能，这样需要进入管理时手动输入管理入口的链接才能显示登录管理界面，不要在前端界面显示管理，这样可以防止随意进入管理功能，因为现在登录管理只需要密码不需要用户名，所以安全性不高，可能被穷举破解，个人建议，希望大神考虑一下，谢谢

**Comment by xhofe** [comment]
前后端分离，不能。
